inc/hero-text-widget.php

```diff
@@ -0,0 +1,253 @@
+<?php
+/**
+ * Hero Text Widget.
+ *
+ * @class      Primer_Hero_Text_Widget
+ * @package    Classes
+ * @subpackage Widgets
+ * @category   Class
+ * @author     GoDaddy
+ * @since      NEXT
+ * @extends    WP_Widget
+ */
+class Primer_Hero_Text_Widget extends WP_Widget {
+
+	/**
+	 * Widget constructor.
+	 */
+	public function __construct() {
+
+		$widget_options = array(
+			'customize_selective_refresh' => true,
+			'classname'                   => 'widget_text primer-widgets primer-hero-text-widget',
+			'description'                 => sprintf(
+				esc_html_x( "A %s theme widget designed for the Hero area on your site's front page.", 'theme name', 'primer' ),
+				esc_html( $this->get_current_theme_name() )
+			),
+		);
+
+		parent::__construct( 'primer-hero-text', esc_html__( 'Hero Text', 'primer' ), $widget_options );
+
+		add_action( 'admin_enqueue_scripts',              array( $this, 'enqueue_scripts' ) );
```

Comment:
@fjarrett Will this only enqueue the script if we are on the widget page or in the customizer?